src/pages/index.js

```diff
@@ -60,16 +165,71 @@ const profilePopup = new PopupWithForm({
 
     formData.input2 = formData.profile_description;
     delete formData.profile_description;
-
-    profileUserData.setUserInfo(formData);
+    api
+      .updateProfileInfo({ name: formData.input1, about: formData.input2 })
+      .then(
+        api.getUserInfo().then((formData) => {
+          formData.input1 = formData.name;
+          formData.input2 = formData.about;
+          profileUserData.setUserInfo(formData);
+        })
+      )
+      .finally(() => {
+        profilePopup.renderLoading(false);
+      });
   },
 });
 
-profilePopup.setEventListeners(); //BULLET POINT #4 RESOLUTION
+api.getUserInfo().then((formData) => {
```

Comment:
Please, don't call `api.getUserInfo()` when you open the profile modal because you have method `getUserInfo` inside `UserInfo` for that not to waste resources on fecthing